docs/reference/cli/options.md

```diff
@@ -56,6 +56,46 @@ WEB3SIGNER_CONFIG_FILE=/home/me/me_node/config.yaml
   </TabItem>
 </Tabs>
 
+### `commit-boost-api-enabled`
+
+<Tabs>
+  <TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--commit-boost-api-enabled[=<BOOLEAN>]
+```
+
+  </TabItem>
+  <TabItem value="Example" label="Example">
+
+```bash
+--commit-boost-api-enabled=true
+```
+
+  </TabItem>
+  <TabItem value="Environment variable" label="Environment variable">
+
+```bash
+TEKU_COMMIT_BOOST_API_ENABLED=true
+```
+
+  </TabItem>
+  <TabItem value="Configuration file" label="Configuration file">
+
+```bash
+commit-boost-api-enabled: true
```

Comment:
The configuration file option should be `eth2.commit-boost-api-enabled: true`. The commit boost cli options are under `eth2` subcommand.